packages/frontend/src/components/Explorer/ExplorerHeader/index.tsx

```diff
@@ -82,6 +83,8 @@ const ExplorerHeader: FC = memo(() => {
                     </Tooltip>
                 )}
 
+                <TimeZonePicker />
```

Comment:
should this be behind a feature flag ?

Reply:
I put the flag in the component so that anywhere we put it it will be hidden. It returns null if the flag is off